scripts/build.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -ev
+
+function initialize {
+  if [ -z "$TLDRHOME" ]; then
+    export TLDRHOME=${TRAVIS_BUILD_DIR:-`pwd`}
+  fi
+  export TLDR_ARCHIVE="tldr.zip"
+  export SITE_HOME="$HOME/site"
+  export SITE_URL="github.com/tldr-pages/tldr-pages.github.io"
+
+  git config --global user.email "travis@travis-ci.org"
+  git config --global user.name "Travis CI"
+  git config --global push.default simple
+  git config --global diff.zip.textconv "unzip -c -a"
+}
+
+function rebuild_index {
+  $TLDRHOME/scripts/build_index.rb
+  echo "Rebuilding index is done"
+}
+
+function build_archive {
+  echo "Removing $TLDR_ARCHIVE if it exists"
+  rm -f $TLDR_ARCHIVE || true
```

Comment:
Having set `set -ev`, is this `|| true` necessary? Wouldn't the script continue executing regardless of the output of `rm`? And wouldn't `rm` not complain if the file doesn't exist because of the `-f` flag?